exonum-light-client/src/main/lombok/com/exonum/client/response/SystemStatistics.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2019 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exonum.client.response;
+
+import lombok.Value;
+
+// todo: I don't like that this class is vaguely defined (just as the endpoint) as SystemStatistics,
+//    because it is unclear what kind of things a user can find inside unless they look here.
+//    A related consideration is if we shall keep ExonumClient#getNumUnconfirmedTransactions
+//    or replace it with ExonumClient#getSystemStats — which does not tell what kind of stats
```

Comment:
I like the approach to slightly redesign the LC and introduce `ExonumClient#getSystemStats`